esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -752,7 +754,9 @@ define(
         Suspend: "Suspender",
         Suspended: "Suspendido",
         SuspendedBy: "Suspendido por",
+        SuspendedByAnyNode: "Suspendido por cualquier nodo",
```

Comment:
same comment as above, but judging by the other "suspended" messages, it seems we're targeting male subject?

Reply:
@miguelvazq can you clarify?

Reply:
`Suspended by any` is the context here.

Reply:
but what is suspended? I suspect the only thing we can suspend are queries

Reply:
> but what is suspended? I suspect the only thing we can suspend are queries

Right. Query suspended by any node

Reply:
> but what is suspended? I suspect the only thing we can suspend are queries
> 
> Right. Query suspended by any node

ok thanks. In that case the suspended messages might need to change to its feminine form

Reply:
ok